### PR TITLE
Update comments for custom rules as requested by reviewer

### PR DESCRIPTION
**Pull request by akhilmohnn**
## Issue

Related to #29012 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by MaoShizhong** [comment]
Carryover from #29012

You don't need to open new PRs for every change requested. PRs just track a branch so you only need to commit and push any changes to your `customRuleComments` branch and the PR will automatically update.

Since you've opened this PR already  I'll use this one and close the old one.

**Comment by akhilmohnn** [comment]
Thank you for your feedback! I’ll go ahead and fix the indentation of the comments as requested. Once I’ve made the changes, I’ll push the updates to this branch.

File: .markdownlint-cli2.jsonc [hunk 1]
```diff
@@ -79,7 +79,9 @@
     // link-fragments
     // Disabled as it uses a different heading conversion algo to the TOP website
     "MD051": false
-  },
+    },
+    // Custom rules specific to the project
+    // Docs for each rule can be found in `./markdownlint/docs`
```
Comment (MaoShizhong) [inline .markdownlint-cli2.jsonc:84]: I meant that the comments were indented too much, not that the closing brace wasn't indented enough.
```suggestion
  },
  // Custom rules specific to the project
  // Docs for each rule can be found in `./markdownlint/docs`
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

File: .markdownlint-cli2.jsonc [hunk 2]
```diff
@@ -79,7 +79,9 @@
     // link-fragments
     // Disabled as it uses a different heading conversion algo to the TOP website
     "MD051": false
-  },
+    },
+  //Custom rules specific to the project
+  //Docs for each rule can be found in `./markdownlint/docs`
```
Comment (MaoShizhong) [inline .markdownlint-cli2.jsonc:84]: The closing brace on line 82 still needs reverting as it was accidentally indented an extra level in a previous commit. Only the comments needed to be moved back with a space after the `//` to be consistent with the other comments.

Comments must share the same indent level as the thing they're attached to, not based on other comments. In this case, the added comments are for the `"customRules"` property so must match its indent level. The other comments are attached to their respective rules and so are indented to match those rules' indent levels.

If it makes it simpler, you can use the "Commit Suggestion" underneath the code suggestion below to commit the necessary changes.
```suggestion
  },
  // Custom rules specific to the project
  // Docs for each rule can be found in `./markdownlint/docs`
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

**Review by MaoShizhong — APPROVED** [review]

**Pull request merged**